aws/src/integration/java/org/apache/iceberg/aws/AwsIntegTestUtil.java

```diff
@@ -127,6 +129,39 @@ public static void cleanS3Bucket(S3Client s3, String bucketName, String prefix)
     }
   }
 
+  public static void cleanS3ExpressBucket(S3Client s3, String bucketName, String prefix) {
+    // For S3 express we just need to delete the objects not the versions.
```

Comment:
1. prefer as a javadoc
2. ideally both this and the general cleanup should delete in progress uploads under the path, so maybe a `abortUploadsUnderPath()` operation should do that and be invoked in both places. saves money everywhere and avoids confusion about deleted paths existing on s3 express

Reply:
I couldn't find anything on abourtUploadsUnderPath() but I might be missing something? Are we sure that this exists in Iceberg?